config/cmake/MFEMConfig.cmake.in

```diff
@@ -11,7 +11,10 @@
 
 include(${CMAKE_CURRENT_LIST_DIR}/MFEMConfigVersion.cmake)
 
+set(MFEM_MPIRUN_CMD @MFEM_MPIRUN_CMD@)
```

Comment:
I don't think we need to have this variable here. CMake has its own variables for this, `MPIEXEC`, `MPIEXEC_NUMPROC_FLAG`, etc; see e.g. `examples/CMakeLists.txt`.